### ISSUE TITLE
Not working right after install

### ISSUE THREAD

**Issue by ckald** [TomAnthony/itermocil#9]
Hi, I installed itermocil, but for any attempt to run a layout, I get the following:

```
➜  ~  itermocil sample
304:311: syntax error: Expected end of line but found class name. (-2741)
```

Debug info (if helps):

```
➜  ~  itermocil sample --debug

tell application "iTerm"
	activate
	delay 0.3
	tell i term application "System Events" to keystroke "t" using command down
	delay 0.3
	tell i term application "System Events" to keystroke "d" using command down
	tell i term application "System Events" to keystroke "]" using command down
	delay 2

	tell twelfth session of current terminal
		write text "cd ~/Code/sample/www; git status"
		set name to "sample-two-panes"
	end tell

	tell thirteenth session of current terminal
		write text "cd ~/Code/sample/www; rails server"
		set name to "sample-two-panes"
	end tell
end tell
```

```
➜  ~  brew info itermocil
tomanthony/brews/itermocil: stable 0.1.5
https://github.com/TomAnthony/itermocil
/usr/local/Cellar/itermocil/0.1.5 (48 files, 744K) *
  Built from source
From: https://github.com/tomanthony/homebrew-brews/blob/master/Formula/itermocil.rb
```

I'm on OS X 10.10.4 with iTerm2 2.1.1

**Comment by rkern** [comment]
I see the same thing with OS X 10.8.5 and iTerm2 2.1.1. The `304:311` in the error message is picking out the `session` in `tell twelfth session of current terminal`. Googling around suggests that perhaps there is a problem with `osascript` [not getting the right dictionary for the application](http://blog.sorrab.com/archives/186), but the suggested fix had no effect for me.

**Comment by TomAnthony** [comment]
Sorry for slow reply on this - I've been away.

I have identified this problem - it was because when there were more than 10 panes open Applescript cannot use the 'fifth' format for numbers that I was using (i.e. it doesn't know what the 'eleventh' pane is), but it does like the '5th' format so I've changed it around in the latest commit.

I'll be doing a release shortly which will contain this fix.
